pages/03.Tutorials/18.create-backup-and-restore/default.en.md

```diff
@@ -68,37 +119,84 @@ $ kubectl delete namespace backup-tutorial
 namespace "backup-tutorial" deleted
 ```
 
-Now we can use the backup to restore the previous state of our application:
+Now we can use the backup to restore the previous state of our deployment:
 
 ```shell
-$ ark restore create --from-backup hello-app-backup-1
-Restore request "hello-app-backup-1-20180813145116" submitted successfully.
-Run `ark restore describe hello-app-backup-1-20180813145116` for more details.
+$ ark restore create --from-backup backup-tutorial
+Restore request "backup-tutorial-20180919163439" submitted successfully.
+Run `ark restore describe backup-tutorial-20180919163439` for more details.
 ```
 
 Ark will now restore the namespace the deployment and the pods running in it:
 
-```shel
-$ kubectl get pods --namespace backup-tutorial
-NAME                         READY     STATUS    RESTARTS   AGE
-hello-app-5c7477d7b7-2qpmm   1/1       Running   0          6s
-hello-app-5c7477d7b7-64psb   1/1       Running   0          6s
+```shell
+$ kubectl -n backup-tutorial get po
+NAME                               READY     STATUS    RESTARTS   AGE
+nginx-deployment-c7df76c4f-bq2zj   1/1       Running   0          29s
+```
+
+Ark will also make sure that your persistent volume is restored if you add the correct annotation to the pod spec:
+
+```shell
+$ kubectl -n backup-tutorial get pvc
+NAME         STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
+nginx-logs   Bound     pvc-24afa31c-bc19-11e8-b699-0a580af40873   1Gi        RWO            sys11-quobyte   1m
+```
```

Comment:
Should we also add something here that shows that actually written data to the volume is the same that was backed up?